### DISCUSSION TITLE
Unable to access microsoft azure free credit

### DISCUSSION THREAD

**Discussion by 20125A0405** [github/india#867] [Students]
I am facing the following issue. Though I am eligible to get free access to Microsoft azure I am unable to claim. When I tried to claim it is showing following messages. 

      **_Looks like you already have an account with us
The email 20125A0405@vidyanikethan.asia in your GitHub profile belongs to your work or school. You cannot use GitHub to sign in.
Choose Next to sign in with your work or school account credentials._**

**Comment by 20125A0405** [comment]
<img width="960" alt="azure2" src="https://user-images.githubusercontent.com/86004724/180701879-0580ae3b-2b63-472d-b779-3c8e31bd69f0.png">
<img width="960" alt="azure1" src="https://user-images.githubusercontent.com/86004724/180701966-7c45ce04-81ed-40db-9e60-bc2fdad0493f.png">

**Reply by jyothibhati** [reply]
got the same problem can i know the solution

**Reply by JayaSaradhiReddy** [reply]
Just clear the browser history, cache and everything from your PC. Try again, prefer to use other browser that is not used to sign up before.

**Comment by Chaitanya800** [comment]
i am not able to access  microsoft azure free account
